package.json

```diff
@@ -322,6 +322,7 @@
     "jest-mock-extended": "1.0.10",
     "jest-transform-stub": "2.0.0",
     "license-checker": "25.0.1",
+    "next-secure-headers": "2.1.0",
```

Comment:
@eirikurn is this something we should take one layer up? Are we doing this somewhere else?

Reply:
Nice PR!

It would be nice to add this to the shared nextjs server for all of our nextjs projects, either with an express middleware or a next config override (since it's tricky to share next.config.js code in another way).

Then it's just the output-static projects that aren't protected.